src/main/target/AT32F435G/target.h

```diff
@@ -34,15 +34,27 @@
 #define HANG_ON_ERRORS
 #endif
 
+#define USE_VCP
+
+#define USE_SOFTSERIAL1
+#define USE_SOFTSERIAL2
+
+#define UNIFIED_SERIAL_PORT_COUNT       3
```

Comment:
UNIFIED_SERIAL_PORT_COUNT is only defined in target.h files so I think it can be removed entirely.

Looks like value was not being evaluated before.